jack.py

```diff
@@ -600,6 +680,14 @@ def transport_locate(self, frame):
         _check(_lib.jack_transport_locate(self._ptr, frame),
                "Error locating JACK transport")
 
+    def transport_query(self):
+        """return current transport position."""
+        # Test rolling transport status
+        if _lib.jack_transport_query(self._ptr, self.position) == 1:
+            return self.position
+        else:
+            return None
```

Comment:
> Yes I think better is to return a tuple ?

Yes, I think returning a tuple would be good.

The question is if we can improve on the numbers in the enum `jack_transport_state_t`.
We can probably wrap this into a custom type that displays something like `jack.ROLLING` ...